[BrowserKit] Add argument $serverParameters to click() and clickLink() (#48841)
symfony/symfony: src/Symfony/Component/BrowserKit/CHANGELOG.md

CHANGELOG
=========

6.3
---

> * Methods `AbstractBrowser::click()` and `AbstractBrowser::clickLink()` will have a new `$serverParameters` argument

Reviewer: ```suggestion
* Added `$serverParameters` argument to methods `AbstractBrowser::click()` and `AbstractBrowser::clickLink()`
```

Author: I did it that way to be consistent with the similar change : 

https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/BrowserKit/CHANGELOG.md#420

>   in version 6.3, not defining it is deprecated

Reviewer: Should the code trigger deprecations when that is the case?

Author: I did it that way to be consistent with a very similar change : https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/BrowserKit/CHANGELOG.md#420
